docs/docs/monitoring/analytics/example-queries.mdx

```diff
@@ -0,0 +1,194 @@
+---
+id: example-queries
+sidebar_label: Example queries
+title: Example queries
+description:
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+
+Below you can find inspiration on how to analyze the conversations your assistant is having.
+Example queries using SQL are provided, together with example visualisation in Metabase.
+
+### Number of sessions per month
+
+It's common to look at high level usage metrics of your chatbot. A metric you could look at is number of
+sessions per month. Here is how it could look like:
+
+
+```sql
+SELECT
+  date_trunc('month', "public"."rasa_session"."timestamp") AS "first_seen",
+  count(*) AS "count"
+FROM "public"."rasa_session"
+GROUP BY 1
+ORDER BY 1 ASC
+```
+
+<div align="center">
+  <img
+    alt="Example result for a query showcasing the number of users per day."
+    src={useBaseUrl("/img/analytics/graph-number-sessions-month.png")}
+    width="100%"
+  />
+</div>
+
+### Number of sessions per channel
+
+If you're connecting your assistant to multiple channels, another metric that could be interesting to look at
+is the number of sessions per channel, let's say per week. You'd get the following query:
```

Comment:
```suggestion
If you're connecting your assistant to multiple channels, it could be useful to look at
the number of sessions per channel, let's say per week. The query you would need for this metric is:
```